### PR TITLE
Luceism Additions

### PR DESCRIPTION
**Pull request by nauticall**
Adds a lot of Luceian stuff for Assunzionis and brings them up to speed as a contender religion.

- Added several items relating to Luceism, the religion of Assunzione.
- Added chaplain items for Luceism: robes, a cassock, and a new null rod, the Luminous Sceptre.
- Added a new versebook, the Luceian Book of Scripture, available to Assunzionii characters. It's a versebook with various texts from Luceism.
- Resprited the Luceian amulet, Luceian Book of Scripture, and the Assunzione cloak.
- Compressed the loadout warding sphere and warding sphere case into just one loadout item that already comes with a case.
- Religions with unique religious books (versebooks) now have their chaplains spawn with the versebook in question, rather than a generic black tome.

image
image
image

### CODE REVIEW

**Comment by nauticall** [comment]
!review

**Comment by sadkermit** [comment]
The dionae lore team aren't fond of the Luceborn origin and accent currently, as there is no explicit lore or named community for dionae on Assunzionii beyond their views on Luceism. That'll have to be gutted from the PR, pretty please.

File: code/modules/background/origins/origins/diona/coalition.dm [hunk 1]
```diff
@@ -2,7 +2,9 @@
 	name = "Coalition of Colonies"
 	desc = "The Coalition of Colonies was born out of the fires of the Interstellar War, the bloodiest war humanity has seen since its dawn as a species. The majority of its citizens prize their freedom above all else, which has led to problems with governance for the Coalition: to this day, it remains a very decentralized and fragmented entity that can only be brought together as a unified front in moments of extreme crisis. But with the recent growth of the Republic of Biesel, retreat of the Solarian Alliance, and an increasingly militaristic Empire of Dominia, perhaps this matter shall change in the years to come."
 	possible_origins = list(
-		/singleton/origin_item/origin/coc_wildborn
+		/singleton/origin_item/origin/coc_grown,
+		/singleton/origin_item/origin/coc_wildborn,
+		/singleton/origin_item/origin/assunzione_dionae ,
```
Comment (listerla) [inline code/modules/background/origins/origins/diona/coalition.dm:7]: This should probably be cut out.

File: html/changelogs/nauticall-luceism_stuff.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rscadd: "Added several items relating to Luceism, the religion of Assunzione."
+  - rscadd: "Added chaplain items for Luceism: robes, a cassock, and a new null rod, the Luminous Sceptre."
+  - rscadd: "Added a new versebook, the Luceian Book of Scripture, available to Assunzionii characters. It's a versebook with various texts from Luceism."
+  - rscadd: "Added a new origin and accent for Dionae, Luceborn, with them originating from Assunzione and being faithful to Luceism."
```
Comment (listerla) [inline html/changelogs/nauticall-luceism_stuff.yml:44]: This too.

File: aurorastation.dme [hunk 3]
```diff
@@ -3673,12 +3674,12 @@
 #include "code\ZAS\Zone.dm"
 #include "code\ze_genesis_call\genesis_call.dm"
 #include "interface\interface.dm"
+#include "interface\skin.dmf"
 #include "interface\fonts\fonts_datum.dm"
 #include "interface\fonts\grand_9k.dm"
 #include "interface\fonts\pixellari.dm"
 #include "interface\fonts\spess_font.dm"
 #include "interface\fonts\tiny_unicode.dm"
-#include "interface\skin.dmf"
```
Comment (NonQueueingMatt) [inline aurorastation.dme:3681]: ?

File: code/modules/clothing/factions/assunzione.dm [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+	name = "luceian cassock"
+	desc = "A linen cassock worn by clergyfolk of the Luceian faith of Assunzione. Despite being of simple make, the violet dye used for the shoulders has an almost iridescent sparkle to them."
+	icon = 'icons/clothing/under/uniforms/assunzione.dmi'
+	icon_state = "assunzione_cassock"
+	item_state = "assunzione_cassock"
+	contained_sprite = TRUE
+	slot_flags = SLOT_ICLOTHING
+
+/obj/item/clothing/suit/storage/hooded/wintercoat/assunzione_robe
+	name = "\improper Luceian Pyramidical keeper robe"
```
Comment (NonQueueingMatt) [inline code/modules/clothing/factions/assunzione.dm:11]: lowercase this

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Geevies — APPROVED** [review]

**Pull request merged**